### PR TITLE
Fix various links and formatting errors

### PR DESCRIPTION
**Pull request by agatemosu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

SKIP_OUTDATED_CHECK
SKIP_WIKILINK_CHECK (just one error that seems like a rabbit hole I don't want to go down here -clayton)

### CODE REVIEW

**Review by cl8n — CHANGES_REQUESTED** [review]
i also dont think any of the text in translations should be changed. which happens in the following files
wiki/Disambiguation/ko.md
wiki/Client/Menu_tips/zh.md
wiki/Client/Beatmap_editor/fr.md

File: wiki/Disambiguation/ko.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ no_native_review: true
 
 [osu!wiki](/wiki/Main_page)에 있는 다양한 **동음이의어**를 다루는 문서입니다.
 
-일반적인 문서에서 제목이 유사한 경우, 반드시 ["다르게도 사용할 수 있습니다" 햇노트](/wiki/Article_styling_criteria/Formatting#다르게도-사용할-수-있습니다)에 동음이의어 페이지 링크를 첨부해야 합니다.
+일반적인 문서에서 제목이 유사한 경우, 반드시 ["다른 의미를 찾으려면" 햇노트](/wiki/Article_styling_criteria/Formatting#다른-의미를-찾으려면)에 동음이의어 페이지 링크를 첨부해야 합니다.
```
Comment (agatemosu) [inline wiki/Disambiguation/ko.md:9]: this refers to a section of the article so should be changed too

File: wiki/Client/Menu_tips/zh.md [hunk 2]
```diff
@@ -82,7 +82,7 @@ tags:
 - 查阅[精选艺术家列表](https://osu.ppy.sh/beatmaps/artists)来获取已经测好时间轴的谱面，并直接开始作图！
 - 把哨、镲和拍手音效放在节奏对应的位置上来润色你的谱面！可以在 `时间轴设定 (Timing Setup)` 窗口 (`F6`) 内的音频 (Audio) 标签页下来修改不同段落的音效组。
 - 将光标悬停在时间轴上，并按住 `Alt`，滚动鼠标滚轮或者按 `+`/`-` 按钮来缩放时间轴。
-- 如果你在作图期间调整时间轴，切记要使用[`重新对齐所有物件`](/wiki/Client/Beatmap_editor/Menu#special-commands.3)功能，重新将所有物件对齐在时间轴上。这个功能位于 `测定时间轴 (Timing)` 选项卡的下拉菜单里。注意未对齐的[继承时间点（绿线）](/wiki/Client/Beatmap_editor/Timing#继承时间点（绿线）)——它们不受这个功能影响！
+- 如果你在作图期间调整时间轴，切记要使用[`全部重新对齐`](/wiki/Client/Beatmap_editor/Menu#测定时间轴-(timing))功能，重新将所有物件对齐在时间轴上。这个功能位于 `测定时间轴 (Timing)` 选项卡的下拉菜单里。注意未对齐的[继承时间点（绿线）](/wiki/Client/Beatmap_editor/Timing#继承时间点（绿线）)——它们不受这个功能影响！
```
Comment (agatemosu) [inline wiki/Client/Menu_tips/zh.md:85]: this is not the editor tip of the localisation in the game, and this refers to the button that does not exist, I only changed the name of the button that it's in game

File: wiki/Client/Beatmap_editor/fr.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Éditeur de Beatmap
+# Éditeur de beatmap
```
Comment (agatemosu) [inline wiki/Client/Beatmap_editor/fr.md:1]: doryanr said that it is correct

**Review by cl8n — APPROVED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**